docs/_static/img/logo-name-medium.svg

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   width="318.67502"
+   height="89.827674"
+   viewBox="0 0 41.983479 11.88345"
+   version="1.1"
+   id="svg5"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg">
+  <defs
+     id="defs130" />
+  <g
+     id="g715"
+     style="fill:#ff0000"
+     transform="translate(-9.9602331,-10.294627)" />
```

Comment:
@jim22k is it safe to remove these lines? Do they do anything? Do they help inkscape in any way? They look delete-able to me.